app/models/whitehall_migration/asset_import.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Represents the raw import of an asset from Whitehall Publisher and
+# the import status of the asset into Content Publisher
+class WhitehallMigration::AssetImport < ApplicationRecord
+  belongs_to :document_import
+  belongs_to :image_revision, class_name: "Image::Revision", optional: true
+  belongs_to :file_attachment_revision, class_name: "FileAttachment::Revision", optional: true
+
+  validate :associated_with_only_image_or_file_attachment
+
```

Comment:
We probably want an enum of states in here that this object can have.

I think you've got not_processed, processing and processed. Is that right? It might be better to start with pending for consistency with DocumentImport. Then I'd suggest having end states of either removed or redirected rather than processed that capture the outcome.